ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -80,17 +83,19 @@ function WorkloadCvesOverviewPage() {
                         Prioritize and manage scanned CVEs across images and deployments
                     </FlexItem>
                 </Flex>
-                <FlexItem className="pf-u-pr-lg">
-                    <Button
-                        variant="secondary"
-                        onClick={() => {
-                            setDefaultWatchedImageName('');
-                            watchedImagesModalToggle.openSelect();
-                        }}
-                    >
-                        Manage watched images
-                    </Button>
-                </FlexItem>
+                {canWriteWatchedImages && (
+                    <FlexItem className="pf-u-pr-lg">
+                        <Button
+                            variant="secondary"
+                            onClick={() => {
+                                setDefaultWatchedImageName('');
+                                watchedImagesModalToggle.openSelect();
```

Comment:
Aha, here is an example where we talked about possibility of a more generic hook.